app/styles/pages/_settings.scss

```diff
@@ -9,7 +9,13 @@
   width: 100%;
   margin-top: -38px;
   margin-bottom: 30px;
-  padding: 30px 0 0px;
+  padding: 30px 0 0;
+
+  @media (max-width: 608px) {
+    padding: 30px 12px 0 12px;
```

Comment:
Too much padding. The existing spacing is already good. Fix the Facebook button instead.